packages/chat-component-bindings/src/chatThreadSelector.ts

```diff
@@ -83,38 +143,41 @@ export const updateMessagesWithAttached = (chatMessagesWithStatus: ChatMessage[]
      * in this case, we only want to show the read statusToRender of last message of the new messages block)
      */
     let attached: boolean | MessageAttachedStatus = false;
+    const previousMessage = messages[index - 1];
```

Comment:
nit: those are constants used only in the 'else' in line 155, why not declare them inside this 'else' instead?
the previousMessage can be anything if index is 0... 
I know it will not be used in case of index being 0, but just don't like the possibility of being used in that case though...
Or can we declare them as undefined if index === 0 for previous and index > messages.length for next?